content/guides/component-lifecycle.mdx

```diff
@@ -6,9 +6,10 @@ These milestones summarize the maturity lifecycle of UI components in Primer. Co
 
 ## Experimental
 
-Proof-of-concept, often built outside of Primer.
+The component exists as a proof-of-concept or is work in progress.
```

Comment:
I wasn't sure how much to write so only added the crucial information. Please feel free to suggest changes.